packaging/macos/Portfile

```diff
@@ -31,12 +31,13 @@ post-fetch {
     system -W ${worksrcpath} "${git.cmd} submodule update --init --recursive"
 }
 
+depends_build            port:npm9 \
```

Comment:
`depends_build` fails, and I don't know why.

https://github.com/LizardByte/Sunshine/blob/a7b0eacecd750ff96d6b526e6ff990a57e0657c6/packaging/macos/Portfile#L34

Reply:
Yep, it worked! I guess macports is creating it's own list so we need to append to it?